flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/ProducerConfigConstants.java

```diff
@@ -24,10 +24,28 @@
  */
 public class ProducerConfigConstants extends AWSConfigConstants {
 
+	/** Deprecated key. **/
+	public static final String DEPRECATED_COLLECTION_MAX_COUNT = "aws.producer.collectionMaxCount";
+
+ 	/** Deprecated key. **/
+	public static final String DEPRECATED_AGGREGATION_MAX_COUNT = "aws.producer.aggregationMaxCount";
+
 	/** Maximum number of items to pack into an PutRecords request. **/
-	public static final String COLLECTION_MAX_COUNT = "aws.producer.collectionMaxCount";
+	public static final String COLLECTION_MAX_COUNT = "CollectionMaxCount";
```

Comment:
Ideally, we should deprecate `ProducerConfigConstants` and remove the class in the long run, since we rely solely on `Properties` parsing now and have no "Flink-specific" key.